e2e_testing/torchscript/xfail_sets.py

```diff
@@ -16,6 +16,8 @@
 COMMON_TORCH_MLIR_LOWERING_XFAILS = {
     "QuantizedMLP_basic",
     "IouOfModule_basic",
+    "SliceEndSleStartModule_basic",
```

Comment:
I meant to mention the issue here so that it's easier to see the reason of expected failure.